app/views/reviews/edit.html.erb

```diff
@@ -9,6 +9,15 @@
       </div>
     </div>
   </div>
+
+  <% if flash[:errors]  %>
+    <% flash[:errors].each do |error| %>
+    <div class="notification is-danger">
+      <!-- <button class="delete"></button> -->
+      <%= error %>
```

Comment:
could add a more descriptive error if you want to. Not necessary though this works just fine.